docs/learn/digital-assets/read-nft-metadata.md

```diff
@@ -0,0 +1,679 @@
+---
+sidebar_label: '🎨 Read LSP8 NFT Metadata'
+sidebar_position: 5
+description: Learn how to read the metadata specific to a LSP8 NFT.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Read LSP8 NFT Metadata
+
+If your digital asset contract is an [LSP8](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) NFT or Collection, you can fetch the metadata of specific token IDs. There are two different ways to retrieve the metadata of a `tokenId` based on the configuration of the contract.
+
+- **call [`getDataForTokenId(...)`](../../contracts/contracts/LSP8IdentifiableDigitalAsset/extensions/LSP8Enumerable.md#getdatafortokenid)**: By using this method, you retrieve the individual metadata directly on the contract. This is the **default and recommended procedure**, as it can be used regardless of if token IDs follow a numeral sequence, are created individually after contract creation, or if the metadata was updated for a certain subset.
+- **read [`LSP8TokenMetadataBaseURI`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenmetadatabaseuri)**: By using this method, you can dynamically generate the URL to individual metadata files. This is usually done when the token IDs of an NFT or Collection can be pre-determined by a sequence, are set up in advance, or if metadata is uploaded or updated in bulk.
```

Comment:
Read LSP8TokenMetadataBaseURI: This method dynamically generates URLs to individual metadata files. It's useful when token IDs in an NFT or collection are pre-determined by a sequence, set up in advance, or if metadata is uploaded or updated in bulk.